packages/@react-spectrum/meter/test/Meter.test.js

```diff
@@ -10,18 +10,14 @@
  * governing permissions and limitations under the License.
  */
 
-import {cleanup, render} from '@testing-library/react';
 import {Meter} from '../';
 import React from 'react';
+import {render} from '@testing-library/react';
 
 describe('Meter', function () {
-  afterEach(() => {
-    cleanup();
-  });
-
   it('handles defaults', function () {
-    let {getByRole} = render(<Meter label="Meter" />);
-    let progressBar = getByRole('meter progressbar');
```

Comment:
Does this no longer work this way? Seems like a good part of the test to make sure the role exists

Reply:
omg lol... of course @devongovett has already found this, hahaha :) https://github.com/testing-library/dom-testing-library/issues/411
I'll update to use what was done in this issue